client/components/marketing-survey/cancel-purchase-form/index.jsx

```diff
@@ -313,10 +365,87 @@ class CancelPurchaseForm extends React.Component {
 		}
 	};
 
+	showUpsell = () => {
+		const { isSubmitting, upsell } = this.state;
+
+		if ( ! upsell ) {
+			return null;
+		}
+
+		const { downgradePlanPrice, purchase, selectedSite, translate } = this.props;
+
+		const dismissUpsell = () => this.setState( { upsell: '' } );
+
+		const Upsell = ( { actionHref, actionText, actionOnClick, children, image } ) => (
+			<div className="cancel-purchase-form__upsell">
+				<img className="cancel-purchase-form__upsell-image" src={ image } alt="" />
+				<div className="cancel-purchase-form__upsell-description">
+					{ children }
+					<GutenbergButton
+						href={ actionHref }
+						isPrimary
+						onClick={ actionOnClick }
+						disabled={ isSubmitting }
+					>
+						{ actionText }
+					</GutenbergButton>
+					<GutenbergButton onClick={ dismissUpsell }>{ translate( 'Dismiss' ) }</GutenbergButton>
+				</div>
+			</div>
+		);
+
+		switch ( upsell ) {
+			case 'business-atomic':
+				return (
+					<Upsell
+						actionOnClick={ this.closeDialog }
+						actionText={ translate( 'Keep my plan' ) }
+						image={ pluginsThemesImage }
+					>
+						<BusinessATStep />
+					</Upsell>
+				);
+			case 'upgrade-atomic':
+				return (
+					<Upsell
+						actionHref={ `/checkout/${ selectedSite.slug }/business?coupon=BIZC25` }
+						actionOnClick={ () =>
+							this.props.recordTracksEvent( 'calypso_cancellation_upgrade_at_step_upgrade_click' )
+						}
+						actionText={ translate( 'Upgrade my site' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **25** times:
`translate( 'Upgrade My Site' )` *ES Score: 9*